templates/thank-you.html

```diff
@@ -46,24 +47,31 @@ <h1 data-qa="title">{{ _("Thank you for completing the {survey_title}").format(
     <a href="https://www.ons.gov.uk/surveys">https://www.ons.gov.uk/surveys</a></p>
   {% endif %}
 
-  {% if content.view_submitted_response.enabled %}
+  {% if content.view_submitted_response.enabled and not content.view_submitted_response.expired %}
     <h1 data-qa="view-submitted-response-title">{{ _("Get a copy of your answers") }}</h1>
-    {% if not content.view_submitted_response.expired %}
-      <p class="ons-u-mb-s">{{ _("You can <a href='{url}' id='view-submitted-response-link'>save or print your answers</a> for your records.").format(url = content.view_submitted_response.url) }}</p>
-      {% call onsPanel({
-        "id": "view-submitted-response-warning",
-        "type": "warn"
-      }) %}
-        <p>{{ _("For security your answers will only be available to view for 45 minutes") }}</p>
-      {% endcall %}
-    {% else %}
-      {% call onsPanel({
+    <p
+      class="ons-u-mb-s">{{ _("You can <a href='{url}' id='view-submitted-response-link'>save or print your answers</a> for your records.").format(url = content.view_submitted_response.url) }}</p>
+
+    {{ onsTimeoutPanel ({
+        "id": "view-submitted-response-countdown",
+        "countdown": {
+            "countdownInSeconds":content.view_submitted_response.countdown_seconds,
+            "minutesTextSingular":"minute",
+            "minutesTextPlural":"minutes",
+            "secondsTextSingular":"second",
+            "secondsTextPlural":"seconds",
+            "countdownText":"For security, your answers will only be available to view for another",
+            "nojsText": "For security, your answers will only be available to view for another " + content.view_submitted_response.countdown_minutes|string + " minutes",
```

Comment:
Added additional property of `countdown_minutes` instead of hard coded 45 minutes. This will be value set `VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS` converted to minutes